.github/workflows/codeql.yml

```diff
@@ -20,47 +20,41 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          path: go/src/github.com/DataDog/datadog-agent
+
```

Comment:
cache go modules
```suggestion

      - name: Cache go modules
        uses: actions/cache@v2
        with:
          path: |
            ~/.cache/go-build
            ~/go/pkg/mod
          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
          restore-keys: |
            ${{ runner.os }}-go-

```

Reply:
Hello ! I think this is a good idea and should absolutely be done (even for Python deps). But currently the go dependencies are downloaded between the codeql init and the codeql analyse steps which means that there is a possibility that they are analysed by CodeQL. I believe this warrants a separate PR to ensure that we are still getting the same amount of security findings.

Reply:
no worries! separated